File: app/src/main/cpp/app/js_compilation_result/JsCompilationResult.cpp
//
// Created by kozlo on 11/19/2023.
//

#include "JsCompilationResult.h"
#include <string>

JsCompilationResult::JsCompilationResult(bool success, std::string error) {
    this->success = success;
    this->error = std::move(error);
}

File: app/src/main/java/com/vengine_android/JsCompilationResult.java
package com.vengine_android;

public class JsCompilationResult {
    private final boolean success;
    private final String error;

    public JsCompilationResult(boolean success, String error) {
        this.success = success;
        this.error = error;
    }

    public boolean isSuccess() {
        return success;
    }

    public String getError() {
        return error;
    }
}
